server.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const fetch = require('node-fetch');
+require('dotenv').config();
+const {configuration, OpenAI } = require ('openai');
+const app = express();
```

Comment:
All the relevant require's needed to enable the page to work. 
Make sure to install express, node fetch, dot env and open ai.